public/static/docs/command-reference/metrics/diff.md

```diff
@@ -0,0 +1,123 @@
+# metrics diff
+
+Show a table of changes in [metrics](/doc/command-reference/metrics#description)
```

Comment:
Just `show changes in ...` ... `table of changes` sounds strange and it's an implementation detail (with some options it might be even not a table)

Reply:
How about

> Show differences among [metrics](/doc/command-reference/metrics#description)
between two versions of the <abbr>DVC repository</abbr>.

? Updated in 1eb88fe0

Reply:
among ... between - can we simplify somehow?

Reply:
Yes I also don't love it, how about "Show metrics differences between two versions of the DVC repository" ?

Reply:
yep, sounds good for me. May be we can do commits here? 

I'm still trying to figure out when version is appropriate. To me it feels like it's fine to use something like _two versions of an experiment_, (e.g. we use `dvc checkout` and `git checkout` to switch between those two versions - when it's clear that those are experiments), _version of a model_, _version of a dataset_. But version of the repo sounds like we have two repos, no?

Not sure about this ^^

commit feels like the safest choice here

Reply:
OK, applied.

> May be we can do commits here? I'm still trying to figure out when version is appropriate.

We had decided to use more general language whenever possible. The `dvc diff` ref. also has the word "versions" for that reason.

> version of the repo sounds like we have two repos, no?

This is a good point though... OK using "commits" here (and in several similar places, see 3eded6d7).

Reply:
I meant 6215bc2c (and https://github.com/iterative/dvc/pull/3308/commits/3eded6d7901cceaa573ad3aa61f00cd02ab1d0ec)

Reply:
On the other hand it compares _workspace_ and the last commit - means that it's not always a commit/commit comparison. 

That's how git handles this:

`git-diff - Show changes between commits, commit and working tree, etc`

Reply:
Yes, you're right again... Let me move this to a new discussion. I think it's the last pending one here 🎉 : https://github.com/iterative/dvc.org/pull/933#pullrequestreview-357958167